README.md

```diff
@@ -1,29 +1,66 @@
+[//]: # "This README.md file is auto-generated, all changes to this file will be lost."
+[//]: # "To regenerate it, use `python -m synthtool`."
 <img src="https://avatars2.githubusercontent.com/u/2810941?v=3&s=96" alt="Google Cloud Platform logo" title="Google Cloud Platform" align="right" height="96" width="96"/>
 
-# @google-cloud/common-paginator
-> A result paging utility used by Google node.js modules
+# [Google Cloud Common Paginator: Node.js Client](https://github.com/googleapis/nodejs-paginator)
 
-Google Cloud Common node.js module contains stuff used by other Cloud API modules.
+[![release level](https://img.shields.io/badge/release%20level-general%20availability%20%28GA%29-brightgreen.svg?style=flat)](https://cloud.google.com/terms/launch-stages)
+[![npm version](https://img.shields.io/npm/v/@google-cloud/paginator.svg)](https://www.npmjs.org/package/@google-cloud/paginator)
+[![codecov](https://img.shields.io/codecov/c/github/googleapis/nodejs-paginator/master.svg?style=flat)](https://codecov.io/gh/googleapis/nodejs-paginator)
 
-* [github.com/googlecloudplatform/google-cloud-node](https://github.com/googlecloudplatform/google-cloud-node)
+
+
+
+A result paging utility used by Google node.js modules
+
+
+
+
+* [github.com/googleapis/nodejs-paginator](https://github.com/googleapis/nodejs-paginator)
 
 Read more about the client libraries for Cloud APIs, including the older
 Google APIs Client Libraries, in [Client Libraries Explained][explained].
 
 [explained]: https://cloud.google.com/apis/docs/client-libraries-explained
 
-### Installing the package
+**Table of contents:**
+
+
+* [Quickstart](#quickstart)
+
+  * [Installing the client library](#installing-the-client-library)
+
+
+* [Versioning](#versioning)
+* [Contributing](#contributing)
+* [License](#license)
+
+## Quickstart
```

Comment:
I think we have an empty `quickstart.js`, perhaps we can just add a tiny bit of sample code on the API here?